src/Command/RoutesCommand.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * This file is part of riesenia/routing package.
+ *
+ * (c) RIESENIA.com
+ */
+declare(strict_types=1);
+
+namespace Riesenia\Routing\Command;
+
+use Cake\Command\Command;
+use Cake\Console\Arguments;
+use Cake\Console\ConsoleIo;
+use Cake\Core\Configure;
+use MixerApi\Core\Utility\NamespaceUtility;
+use Riesenia\Routing\Attributes\Resource;
+use Riesenia\Routing\Attributes\Route;
+
+class RoutesCommand extends Command
+{
+    /**
+     * @var array<string,array<\Riesenia\Routing\Attributes\Resource|\Riesenia\Routing\Attributes\Route>>
```

Comment:
1. why use FQDN here? 
2. what's the case for `string` here? doesn't look from `addRoute` that this should be possible

Reply:
string is the key, which is the scope+plugin

Reply:
also i use FQDN for Resource class so fixer doesn't change it to resource

Reply:
> string is the key, which is the scope+plugin

but how would it get into this array? it's protected and you write to it using `addRoute` method, which is strongly typed to `Resource|Route`

Reply:
i have shared params, scope and plugin in the resource and route,under attribute class,
that define a default value, empty plugin and scope '/'
so we call the parent method getScope and getPlugin.